ui/de-lib/src/main/java/org/iplantc/de/client/DEClientConstants.properties

```diff
@@ -2,4 +2,7 @@ fileUploadServlet = /de/secured/fileUpload
 fileDownloadServlet = /de/secured/fileDownload
 logoutUrl = j_spring_security_logout
 deHelpFile = https://pods.iplantcollaborative.org/wiki/display/DEmanual/
+notificationWS = /de/websocket/notifications
+sysmessageWS = /de/websocket/system-messages
+wsProtocol = ws://
```

Comment:
My reasoning was a little more selfish: it will allow us to avoid adding a new configuration parameter, which will make testing the DE on our workstations easier. :)